src/Functions/toDaysSinceYearZero.cpp

```diff
@@ -11,13 +11,43 @@ using FunctionToDaysSinceYearZero = FunctionDateOrDateTimeToSomething<DataTypeUI
 
 REGISTER_FUNCTION(ToDaysSinceYearZero)
 {
-    factory.registerFunction<FunctionToDaysSinceYearZero>(FunctionDocumentation{
-        .description = R"(
-Returns for a given date or date with time, the number of days passed since 1 January 0000 in the proleptic Gregorian calendar defined by ISO 8601.
-The calculation is the same as in MySQL's TO_DAYS() function.
-)",
-        .examples{{"typical", "SELECT toDaysSinceYearZero(toDate('2023-09-08'))", "713569"}},
-        .category = FunctionDocumentation::Category::DateAndTime});
+    FunctionDocumentation::Description description_toDaysSinceYearZero = R"(
```

Comment:
Please remove suffixes `_toDaysSinceYearZero` as much as possible. Applies to here but also other functions.